src/Infrastructure/Services/MailService.cs

```diff
@@ -0,0 +1,58 @@
+using MailKit.Net.Smtp;
+using MimeKit;
+using MimeKit.Text;
+using System.Threading.Tasks;
+using System.Web;
+using System;
+
+using Codidact.Authentication.Application.Common.Interfaces;
+using Codidact.Authentication.Domain.Entities;
+using Codidact.Authentication.Application.Options;
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Codidact.Authentication.Infrastructure.Services
+{
+    public class MailService : IMailService
+    {
+        private MailOptions _emailConfiguration;
+        private IServiceProvider _secretsService;
+        public MailService(MailOptions emailConfiguration, IServiceProvider secretsService)
+        {
+            _emailConfiguration = emailConfiguration;
+            _secretsService = secretsService;
+        }
+        public async Task SendEmailAsync(ApplicationUser user, string subject, string textMessage)
+        {
+            var message = new MimeMessage();
+            message.From.Add(new MailboxAddress(_emailConfiguration.SenderName, _emailConfiguration.Sender));
+            message.To.Add(new MailboxAddress(user.UserName, user.Email));
+            message.Subject = subject;
+
+            message.Body = new TextPart(TextFormat.Html)
```

Comment:
This is marked as HTML but the parameter is called text.

Reply:
Since you call this method with HTML you can simply rename the parameter to something like `htmlMessage` then it's less confusing.